overlays/aws-overlay.yaml

```diff
@@ -7,6 +7,13 @@ applications:
     charm: aws-integrator
     num_units: 1
     trust: true
+  aws-cloud-provider:
+    charm: aws-cloud-provider
+
 relations:
-  - ['aws-integrator', 'kubernetes-control-plane']
-  - ['aws-integrator', 'kubernetes-worker']
+  - ["aws-integrator", "kubernetes-control-plane"]
+  - ["aws-integrator", "kubernetes-worker"]
```

Comment:
```suggestion
```
we can drop these, right?

Reply:
Nope, can't drop these sadly.  They're still used here to appropriate tag machines and apply IAM policies